MachineTasks[] all_machine_tasks
TransporterTasks[] all_transporter_tasks